app/assets/javascripts/models/beta/note.js

```diff
@@ -15,6 +15,7 @@ export const post = (projectId, storyId, note) =>
 export const addNote = (story, note) => (
   {
     ...story,
+    _editing: {...story._editing},
```

Comment:
Why not `_editing: story._editing,`? 🤔

Reply:
Actually this part is redundant, I'm removing it now.